app/models/course/assessment/question/programming.rb

```diff
@@ -110,22 +110,37 @@ def non_autograded_template_files=(template_files)
 
   private
 
+  # Create new package or re-evaluate the old package.
+  def process_package
+    if attachment_changed?
+      attachment ? process_new_package : remove_old_package
+    elsif time_limit_changed? || memory_limit_changed? || language_id_changed?
```

Comment:
Ok, so for now if someone changes it from say Python 3.5 to some other language (eg. Javascript), do we want the process_package to run?